hab-pkg-deb/plan.sh

```diff
@@ -0,0 +1,80 @@
+# This file is the heart of your application's habitat.
+# See full docs at https://www.habitat.sh/docs/reference/plan-syntax/
+
+pkg_name=hab-pkg-deb
+pkg_origin=yzl
```

Comment:
Since this if chef/habitat-exporters, should this be `chef`? When building, your `$HAB_ORIGIN` will automatically override this if set.